### PR TITLE
feat(webapp): instance-admin console with overview dashboard and emergency silent mode

### LINKED ISSUE #1386: feat(webapp): instance-admin console shell — overview dashboard and grouped navigation
**Shipped in #1393 (2026-08-03):** `/admin` now has an overview dashboard, grouped Access/Detection/AI/Operations navigation, defined empty states, and an instance-settings surface. The merge also delivered the first partial Silent Mode implementation. The console scope in this issue is complete; the fail-safe defaults, enforced egress gateway/ArchUnit inventory, cross-lane suppression, and full-cycle proof remain tracked only in #1355.

## Current state and merge coordination

#1393 merged on top of #1425/#1426 and kept the Catalog discoverable under Detection. The overview reports delivery state, workspace/membership/login-provider counts, and recent authentication activity. Silent Mode is deliberately not considered complete by this closed console issue; see #1355.

## Problem

The instance-admin area is a flat 4-item nav (Users, Workspaces, Login providers, Audit log) with no index page (`/admin` renders a bare outlet). The 1.0 backlog adds ~6 instance surfaces that currently have **no navigation home at all**: silent-mode brake + channel force-offs (#1357), curated catalog (#1361), labeling queue (#1365), cost/usage (#1368), sync status and background-job health. Without a shell, each lands as an orphan page.

## What

- **`/admin` index = instance overview dashboard**: silent-mode state banner, today's spend, job/sync health at a glance, workspace count — the operator's "is everything fine" page.
- **Grouped navigation** replacing the flat list (e.g. *Access*: users, workspaces, login providers · *Detection*: catalog, labeling, evaluation · *Operations*: audit, sync & jobs, usage & costs, instance settings). Two-level instance-vs-workspace split stays hard — never interleave (the universal pattern: GitLab Admin Area, [Sentry org vs project settings](https://docs.sentry.io/organization/)).
- Instance settings page seeded with the emergency silent-mode control (asymmetric friction: engaging the brake = one click + reason; releasing it = heavy confirm — cheap to silence, expensive to un-silence, per [NN/g proportional-friction guidance](https://www.nngroup.com/articles/confirmation-dialog/)).
- Every list/panel ships a defined empty state: status text + one CTA ([NN/g](https://www.nngroup.com/articles/empty-state-interface-design/)).

## Non-goals

Admin-wide search (revisit when >2 settings pages — [Discourse's trajectory](https://meta.discourse.org/t/introducing-comprehensive-admin-search/360157)); theming; per-admin customization.

## Dependencies

Enabler for the instance parts of #1357, #1361, #1365, #1368 and the sync/jobs UI. Files: `webapp/src/components/core/sidebar/NavSuperAdmin.tsx`, `webapp/src/routes/_authenticated/admin.tsx`.

Part of #1354.

### PR DESCRIPTION
**Pull request by FelixTJDietrich**
## Description

The instance-admin area was a flat four-item nav (Users, Workspaces, Login providers, Audit log) with no index page — `/admin` rendered a bare outlet. The 1.0 backlog adds roughly six more instance surfaces (silent-mode brake, curated catalog, labeling queue, cost/usage, sync status, job health) that currently have **no navigation home at all**, so each would land as an orphan page.

This PR builds the shell they can hang off, and implements the first control that lives in it.

**The console shell**

- **`/admin` is now an overview dashboard** — the operator's "is everything fine" page: delivery state, workspace/membership/login-provider counts, and recent auth activity. Every tile links to the page that manages it.
- **Grouped navigation** (*Access* · *Detection* · *AI* · *Operations*) replaces the flat list, absorbing the AI models/usage pages from #1400 and the instance practice catalog from #1425 — the *Detection* group the issue asked for now has its first occupant. The instance-vs-workspace split stays hard — the two levels are never interleaved. Labeling and evaluation join *Detection* as they land; each is one entry in `ADMIN_NAV_GROUPS`.
- **A new instance settings page**, seeded with the emergency silent-mode control.
- Every new list/panel ships a defined empty state (status text + one CTA).

**Silent mode is real, not just rendered**

A UI-only brake would be worse than none, so this enforces it server-side. A singleton `instance_settings` row backs `GET /admin/settings` and `PUT /admin/settings/silent-mode` (`app_admin` only), and a `core.settings.spi.SilentModeQuery` read port is consulted at **every outbound choke point**:

| Surface | Where it's enforced |
| --- | --- |
| Practice feedback on PRs/MRs **and** issues | `PracticeFeedbackDeliveryPolicy` — the single delivery-decision point #1426 introduced, so the brake sits alongside artifact-closed / merged / draft / opted-out and returns the same `Decision.suppressed(…)` |
| Slack messages, DMs, mentor stream opens | `SlackMessageService` |
| GitLab `/hephaestus review` ack reaction | `BotCommandProcessor` |

Engaging is one click plus an optional reason. Releasing takes a type-to-confirm dialog — widening blast radius deserves more friction than narrowing it ([NN/g](https://www.nngroup.com/articles/confirmation-dialog/)). While engaged, a banner is pinned across every console page.

**What silent mode deliberately does *not* do** (called out because a safety control's docs must be exact):

- **It suppresses outbound delivery only.** Reviews still run and their findings are still saved — they're only held back from posting, and recorded as withheld with the reason `INSTANCE_SILENCED`. Compute/LLM spend continues.
- **Suppression is not a backlog.** Held-back feedback is *not* auto-posted on release. The shadow-backlog + replay flow is #1357/#1358.
- **It's checked once at the entry of each delivery**, so a delivery already in flight completes rather than tearing off half-posted. Same contract as the in-flight Slack stream exemption.
- **Reads are uncached on purpose.** A cache would add staleness in the unsafe direction (still sending just after an engage). A DB failure on the read therefore fails *closed* — the correct direction for a brake.

**Also in here**

- **A `.gitignore` fix worth flagging**: the rule `instance/`, inherited from a Flask template, silently ignored the entire new `webapp/src/components/admin/instance/` directory. Narrowed to `/instance/` (root-anchored, which is what the Flask rule actually meant). Worth a look — any future webapp directory named `instance/` would have been invisible to git.
- `SEVERITY_DOT` is hoisted out of `AdminAuditTable` into the shared `auditFormat.ts` so the overview and the audit table can't drift.
- `admin.login-providers` widened `max-w-5xl` → `max-w-6xl` to match its siblings, now that a global banner sits above it.

**Notes for reviewers**

- The "Memberships" tile sums per-workspace membership counts, so a user in N workspaces counts N times. It's labelled *Memberships* (not *Members*) on purpose — there's no instance-wide distinct-account endpoint yet, and a headline number that disagrees with the page it links to is worse than an honest one.
- Silent mode records a `SUPPRESSED` ledger unit carrying the new `FeedbackSuppressionReason.INSTANCE_SILENCED`, alongside the artifact-closed / merged / draft / opted-out reasons. That satisfies the evaluation contract #1402 introduced — a withheld unit must say *why*, or an evaluator cannot tell "the model missed it" from "policy withheld it". Recording it is not the same as replaying it: released feedback is still not auto-posted (#1357/#1358).
- The toggle itself is audited. #1401's audit-by-default rule requires every admin mutation endpoint to declare its audit status, so `PUT /admin/settings/silent-mode` records `SILENT_MODE_CHANGED` on the `auth_event` trail — instance-level, hence not `config_audit_event`, whose `workspace_id` is `NOT NULL`.
- `InstanceSettingsService.get()` self-heals the singleton row via `INSERT … ON CONFLICT DO NOTHING`. Production seeds it via Liquibase; the self-heal exists for schemas built without migrations (tests run `ddl-auto: create`). The upsert is deliberate — a plain `save()` would PK-violate and poison the surrounding transaction on Postgres under a race.

Fixes #1386

## How to test

**Automated** — `pnpm run check`, then:
- `cd server && ./mvnw test -Dsurefire.includedGroups=unit` (5858 pass)
- `cd server && ./mvnw test -Parchitecture-tests` (224 pass)
- `cd webapp && pnpm run test` (782 pass) and `pnpm run test:storybook src/components/admin/instance` (14 pass)

**Manually**, signed in as an `APP_ADMIN`:
1. Click **Instance admin** in the sidebar footer → lands on the new `/admin` overview with grouped nav.
2. On **Instance settings**, engage silent mode with a reason → a red banner appears on every admin page and the overview's Delivery tile flips to "Silent mode engaged".
3. While engaged, comment `/hephaestus review` on a GitLab MR → **no 👀 reaction** appears and **no feedback posts**. Server logs show `Delivery suppressed: instance silent mode engaged`. A Slack send fails with `silent_mode_engaged`.
4. Release it → the confirm button stays disabled until you type `release`. Delivery resumes.
5. Break `GET /admin/settings` (stop the DB) and reload `/admin/settings` → you get an error + Retry, **not** a skeleton that hides the control.

**Migration**: `1785678171817_changelog.xml` creates `instance_settings` and seeds one row (brake released). It also widens `chk_feedback_suppression_reason` for the new value. Verified with `mvn liquibase:update` against a throwaway Postgres — all 876 changesets apply, the seed row reads back, and the schema-drift gate is clean. Re-running is a no-op (both changesets are preconditioned).

## Screenshots

Rendered from the Storybook stories that ship with this PR (`components/admin/instance/*`); Chromatic diffs them in CI. Images live on the [`assets/pr-1393`](https://github.com/ls1intum/Hephaestus/tree/assets/pr-1393) orphan branch so no binaries land in the source history.

**The instance settings control — asymmetric friction.** Engaging is one click plus an optional reason; releasing needs type-to-confirm.

| Released | Engaged |
| --- | --- |
| Silent mode card in the released state, showing a green Released badge, who last changed it, and an 'Engage silent mode…' button | Silent mode card in the engaged state, showing a red Engaged badge, who engaged it and when, the reason, and a 'Release silent mode…' button |

**Pinned across every console page while engaged** — an engaged brake must be impossible to miss.

Red banner reading 'Silent mode is engaged — nothing is being delivered', naming who engaged it, how long ago, and the reason, with a Manage button

**On the `/admin` overview.**

| Delivery state tile | Recent activity |
| --- | --- |
| Overview Delivery tile showing a red 'Silent mode engaged' badge and who engaged it | Recent activity card listing latest auth events with severity dots and relative timestamps, linking to the full audit log |

## Summary by CodeRabbit

* **New Features**
  * Added an instance-admin overview dashboard with workspace, membership, delivery, and recent authentication activity summaries.
  * Added instance-wide Silent Mode controls with optional reasons, status metadata, and confirmation safeguards when releasing.
  * Silent Mode suppresses outbound feedback, Slack messages, and command acknowledgement reactions while preserving saved findings.
  * Added a persistent admin banner and grouped navigation for settings and management pages.
  * Silent Mode changes are recorded in the audit log.

* **Documentation**
  * Documented the admin console and Silent Mode behavior.

### CODE REVIEW

File: server/src/main/java/de/tum/cit/aet/hephaestus/agent/handler/FeedbackDeliveryService.java [hunk 1]
```diff
@@ -137,6 +141,16 @@ private void doDeliver(AgentJob job, @Nullable DeliveryContent delivery, @Nullab
...
         // Suppression guards
 
+        // Instance-wide emergency brake (#1386) — checked once at entry, before per-workspace/per-user
+        // settings, so an engaged brake suppresses this PR/MR feedback delivery. A delivery already past
+        // this point completes its summary+notes rather than tearing off half-posted (same "in-flight
+        // finishes" contract as the Slack stream exemption). Issue feedback and Slack are guarded at their
+        // own choke points: IssueReviewHandler and SlackMessageService.
+        if (silentModeQuery.isSilentModeEngaged()) {
+            log.warn("Delivery suppressed: instance silent mode engaged, jobId={}", job.getId());
+            return;
```
Comment (ValentinGruener) [inline server/src/main/java/de/tum/cit/aet/hephaestus/agent/handler/FeedbackDeliveryService.java]: Sonst sehe ich selten einfach nur ein leeres return. Normalerweise steht im Code etwas wie recordGateSuppressed. Ich denke, für das Logging wäre so etwas auch im Silent Mode sinnvoll.

File: server/src/main/java/de/tum/cit/aet/hephaestus/agent/handler/IssueReviewHandler.java [hunk 2]
```diff
@@ -212,6 +216,10 @@ void postIssueNote(AgentJob job, PracticeDetectionResultParser.@Nullable Deliver
         if (delivery == null || delivery.mrNote() == null) {
             return;
         }
+        if (silentModeQuery.isSilentModeEngaged()) {
+            log.warn("Issue delivery suppressed: instance silent mode engaged, jobId={}", job.getId());
+            return;
+        }
```
Comment (ValentinGruener) [inline server/src/main/java/de/tum/cit/aet/hephaestus/agent/handler/IssueReviewHandler.java]: Sonst sehe ich selten einfach nur ein leeres return. Normalerweise steht im Code etwas wie recordGateSuppressed. Ich denke, für das Logging wäre so etwas auch im Silent Mode sinnvoll.

**Comment by FelixTJDietrich** [comment]
## Maintainer direction after #1423

Keep this PR, but narrow it to **#1386: the instance-admin console shell**.

During the rebase onto current `main`:

- Keep the overview shell, grouped instance navigation, honest empty states, and reusable admin layout.
- Preserve #1423's route-boundary page layout, navigation semantics, generated route tree, and workspace Review settings.
- Remove the current silent-mode implementation from this PR: `instance_settings` changelog/entity/API, `SilentModeQuery`, delivery/Slack/bot-command call-site checks, `INSTANCE_SILENCED`, silent-mode UI/banner, and delivery-state overview tile.
- Regenerate OpenAPI/client/routes instead of hand-merging generated files.
- Retitle/update the body and keep `Closes #1386`; do not claim #1355.

Why split: the present silent mode defaults released and inventories selected call sites rather than enforcing #1355's gateway/ArchUnit contract. After this shell merges, #1355 gets a fresh branch with safe upgrade defaults, gateway enforcement, declared exemptions, and full-cycle zero-egress coverage.

**Comment by FelixTJDietrich** [comment]
## Updated maintainer direction — supersedes the split comment above

Review size and schema breadth are accepted. Rework this PR **in place** as the combined implementation of #1386 and #1355.

Sequence: merge #1419 first, then rebase this branch onto current `main`.

Keep:
- console overview/grouped navigation and instance-settings UI;
- the `instance_settings` persistence/API foundation;
- asymmetric engage/release UX and prominent banner.

Change before readiness:
- fresh install, upgrade migration, and absent-row fallback must all fail safe to **engaged**;
- replace selected call-site checks with #1355's enforced egress gateway;
- add the SPI/client-surface ArchUnit inventory and declared exemption allowlist;
- cover SCM summary/inline/approval, Slack messages/nudges/digests, and conversational rerouting;
- preserve prospective-only release and live-delivery edit semantics;
- add a full-cycle integration test proving observations + ledger entries persist with zero external writes;
- preserve #1423's route/layout/navigation semantics and regenerate generated artifacts.

When complete, update title/body and close both #1386 and #1355. Keep the PR draft during the rework.
